.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.4"
+          coverage: none
+          tools: composer
+
+      - name: Install dependencies
+        run: |
+          cd wp-content/themes/dxw-security-2017
+          composer --no-interaction install
+
+      - name: Run linter
+        run: |
+          vendor/bin/php-cs-fixer fix --dry-run -v --diff
```

Comment:
These following commands assume that we're in the `wp-content/themes/dxw-security-2017`. It might be better to run `composer install -d wp-content/themes/dxw-security-2017 --no-interaction` and use the full paths in the following scripts. Or, better still, use a `script/cibuild` script to rule them all.

Reply:
Trying to do the minimum here, but yes, already discovered that :).

Reply:
I'm also not really a fan of the `cibuild` script. I'd rather have broken down steps in my CI config than a script that lumps them all together.